test/hotspot/jtreg/vmTestbase/nsk/share/runner/RunParams.java

```diff
@@ -225,7 +225,7 @@ else if (args[i].equals("-iterations"))
                                 iterations = Integer.parseInt(args[++i]);
                 }
                 // Allow to force using vthreads using wrapper property
-                if(System.getProperty("main.wrapper") != null && System.getProperty("main.wrapper").equals("Virtual")) {
+                if(System.getProperty("test.thread.factory") != null && System.getProperty("test.thread.factory").equals("Virtual")) {
```

Comment:
I think there are better examples of how to do this in the diffs above. Maybe a good time to clean it up.